pkg/dashboard/worker/gitops.go

```diff
@@ -121,15 +111,16 @@ func processEvent(
 		token, _, _ = tokenManager.Token()
 	}
 
+	eventRepo, _ := repoCache.InstanceForRead(event.Repository)
```

Comment:
I don't think you can use this field like that.

Our model sucks, and it is not obvious, but this is only set for artifact events, not for release request events.
We only use it to search fro artifacts. (It has a comment: denormalized something something)

long story short, it will break if you push the deploy button on the gui.